backend/open_webui/routers/images.py

```
@@ -419,15 +429,15 @@ def load_b64_image_data(b64_str):
     try:
         if "," in b64_str:
             header, encoded = b64_str.split(",", 1)
-            mime_type = header.split(";")[0]
+            mime_type = header.split(";")[0].lstrip("data:")
```

Ruff (0.12.2)

Using `.strip()` with multi-character strings is misleading

(B005)